### PR TITLE
New folder piero

### PR DESCRIPTION
**Pull request by pierogio**

### CODE REVIEW

File: alumnos/piero/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM python:3.7-alpine
+
+
+RUN mkdir -p /mnt/
+WORKDIR /mnt/
```
Comment (roland1308) [inline alumnos/piero/Dockerfile]: WORKDIR should be go in the second line and should be WORKDIR /app

File: alumnos/piero/service.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+help() {
+   echo "Usage: $0 {start|help}"
+}
+
+
+case "$1" in
+start)
+start
+;;
+*)
```
Comment (roland1308) [inline alumnos/piero/service.sh]: As per instructions, there should be present also the option "HELP"

**Review by roland1308 — APPROVED** [review]
Good job, issues are not bad :-)

File: alumnos/piero/service.sh (hunk 2 above)
Comment (pierogio) [reply alumnos/piero/service.sh]: ty :)

**Review by ricardozd — APPROVED** [review]

**Pull request merged**